README.md

```diff
@@ -193,6 +197,20 @@ This is how you actually render the React components you exposed to `window` ins
 
 This is a helper method that takes any JavaScript expression and returns the output from evaluating it. If you have more than one line that needs to be executed, wrap it in an IIFE. JS exceptions will be caught and console messages handled properly.
 
+### Redux
+See examples of using a generator function to create the top level react-redux component.
+
+### React-Router
+There's no special instructions needed for react-router for client rendering.
+
+If you're server rendering with react-router, you'll need to use the option generator_function to return either:
+
+1. React Component
+2. Object (which we'll call the `routerResult`) with either the shape of:
+  `{ error, redirectLocation }`
+
+You should consult the docs for [react-router server rendering](https://github.com/rackt/react-router/blob/master/docs/guides/advanced/ServerRendering.md) as well as the example in the [React Webpack Rails Tutorial Code](https://github.com/shakacode/react-webpack-rails-tutorial) for how to do this.
```

Comment:
If "do this" is referring to passing the `generator_function: true` option, we explain this in this repo's README, so we should probably just either leave this out or use an internal link so that people don't have to leave the repo to figure out how to do this.